### PR TITLE
Glyphicon not in line with the message #7195

### LINKED ISSUE #7195: Glyphicon not in line with the message
not

### PR DESCRIPTION
**Pull request by Mynk96**
Fixes #7195

### CODE REVIEW

**Comment by Mynk96** [comment]
@whipermr5 Please review!

**Comment by chowyb** [comment]
@Mynk96 I'm not reviewing this but as a note, any PRs involving UI changes should provide a screenshot of the new UI, even though it's a rather minor change in this case.

**Comment by Mynk96** [comment]
@chowyb thank you for mentioning, I should have uploaded the screenshot.
Here's the screenshot for the change:-
changed

File: src/main/webapp/jsp/instructorCourseEnrollResult.jsp [hunk 1]
```diff
@@ -19,8 +19,8 @@
 </c:set>
 
 <ti:instructorPage pageTitle="TEAMMATES - Instructor" bodyTitle="Enrollment Results for ${data.courseId}" jsIncludes="${jsIncludes}">
-    <div class="alert alert-success">
-        <form name='goBack' action="${data.instructorCourseEnrollLink}" method="post" role="form">
```
Comment (whipermr5) [inline src/main/webapp/jsp/instructorCourseEnrollResult.jsp]: Would it work if you move the form outside the div instead? The alert classes are meant to style divs. I.e. `<form><div>...</div></form>`

Comment (Mynk96) [inline src/main/webapp/jsp/instructorCourseEnrollResult.jsp]: `<form name='goBack' action="${data.instructorCourseEnrollLink}" method="post" role="form" > 
            <div class="alert alert-success">
				Enrollment Successful. Summary given below. Click <a id="edit_enroll" href="javascript:document.forms['goBack'].submit()">here</a> to do further changes to the student list.
				<input type="hidden" name="${COURSE_ID}" value="${data.courseId}">
				<input type="hidden" name="${STUDENTS_ENROLLMENT_INFO}" value="${fn:escapeXml(data.enrollStudents)}">
			</div>
		</form>`
this is working..

Comment (Mynk96) [inline src/main/webapp/jsp/instructorCourseEnrollResult.jsp]: I'll commit the changes!

**Comment by Mynk96** [comment]
@whipermr5 We have two more pages with similar issue i.e filterPanel.tag in admin activity and email tab.
prob
and I think we should use alert-info there instead of alert-success as we are just displaying the information or we can have only one filter glyphicon and no ok-sign glyphicon before that.
sol

**Comment by whipermr5** [comment]
@Mynk96 Good job! Can you fix those too? The [`p`](https://github.com/TEAMMATES/teammates/blob/e30eac4dba237177851d02d7b8944f8e1199ece5/src/main/webapp/WEB-INF/tags/admin/activity/filterPanel.tag#L54) looks redundant as the `text-center` class can be applied to the `div` itself.

I agree that `alert-info` would be more appropriate, but that's an unrelated issue, so don't change it yet (maybe you could open a `d.FirstTimers` issue for it).

**Comment by Mynk96** [comment]
@whipermr5 I think it would be fine if I make changes (applying text-center class to div) in this PR only?

**Comment by whipermr5** [comment]
@Mynk96 Yup! That's still related to the issue so it's fine.

**Comment by Mynk96** [comment]
@whipermr5 done!

**Pull request merged**